packages/rollup-full-node/src/app/web3-rpc-handler.ts

```diff
@@ -378,6 +416,7 @@ export class DefaultWeb3Handler implements Web3Handler, FullnodeHandler {
   }
 
   public async sendRawTransaction(rawOvmTx: string): Promise<string> {
+    const timestamp = getTimestamp()
```

Comment:
we need to handle `sendRawTransaction` differently in the test rpc handler, setting timestamp to `utils.getTimestamp + timestampIncreaseSeconds`